contracts/contracts/vault/VaultCore.sol

```diff
@@ -671,6 +659,24 @@ contract VaultCore is VaultStorage {
         return assets[_asset].isSupported;
     }
 
+    function _toUnits(uint256 raw, address _asset)
```

Comment:
I love how this function simplifies a bunch of the code above. 💪 
nit: would add documentation to it